starfish/image/_stack.py

```diff
@@ -8,6 +8,13 @@
 
 
 class ImageStack(ImageBase):
+    INDEX_MAP = {
+        Indices.HYB: 0,
+        Indices.CH: 1,
+        Indices.Z: 2,
+    }
+    MAX_INDEX = max(INDEX_MAP.values()) + 1
```

Comment:
maybe `N_INDICES` ? I think of max_index as the largest dimension.

Reply:
will fix.

Reply:
Changed.